src/store.ts

```diff
@@ -131,19 +145,25 @@ class MSSQLStore extends ExpressSessionStore implements MSSQLStoreDef {
    */
   // ////////////////////////////////////////////////////////////////
   private async dbReadyCheck() {
-    try {
-      if (!this.databaseConnection.connected && !this.databaseConnection.connecting) {
-        await this.initializeDatabase();
-      }
+    for (let attempt = 1; attempt <= (this.retries + 1); attempt++) {
```

Comment:
I forgot how strict eslint is in this repo. 

```suggestion
    for (let attempt = 1; attempt <= this.retries + 1; attempt += 1) {
```